packages/synapse-react-client/src/style/components/query_filter/_enum-facet-filter.scss

```diff
@@ -16,12 +16,15 @@
     .EnumFacetFilter__checkboxContainer {
       cursor: pointer;
     }
+    .EnumFacetFilter__noMatch,
+    .EnumFacetFilter__checkboxContainer,
+    .EnumFacetFilter__checkboxContainer--forAll {
+      padding: 2px 16px;
+    }
```

Comment:
tweaking the EnumFacetFilter dropdown menu padding just so it looks a bit cleaner